src/DssSpell.sol

```diff
@@ -52,157 +67,124 @@ contract DssSpellAction is DssAction {
-        DssExecLib.sendPaymentFromSurplusBuffer(BLOCKCHAINCOLUMBIA,   2_851);
-        // Frontier Research LLC - 2285 DAI - 0xA2d55b89654079987CF3985aEff5A7Bd44DA15A8
-        DssExecLib.sendPaymentFromSurplusBuffer(FRONTIERRESEARCH,     2_285);
-        // Chris Blec - 1334 DAI - 0xa3f0AbB4Ba74512b5a736C5759446e9B50FDA170
-        DssExecLib.sendPaymentFromSurplusBuffer(CHRISBLEC,            1_334);
-        // CodeKnight - 355 DAI - 0x46dFcBc2aFD5DD8789Ef0737fEdb03489D33c428
-        DssExecLib.sendPaymentFromSurplusBuffer(CODEKNIGHT,             355);
-        // ONESTONE - 342 DAI - 0x4eFb12d515801eCfa3Be456B5F348D3CD68f9E8a
-        DssExecLib.sendPaymentFromSurplusBuffer(ONESTONE,               342);
-        // pvl - 56 DAI - 0x6ebB1A9031177208A4CA50164206BF2Fa5ff7416
-        DssExecLib.sendPaymentFromSurplusBuffer(PVL,                     56);
-        // ConsenSys - 33 DAI - 0xE78658A8acfE982Fde841abb008e57e6545e38b3
-        DssExecLib.sendPaymentFromSurplusBuffer(CONSENSYS,               33);
+        // MOMC Parameter Changes
+        // https://vote.makerdao.com/polling/QmYUi9Tk
+
+        // Increase WSTETH-B Stability Fee to 0.25%
+        DssExecLib.setIlkStabilityFee("WSTETH-B", ZERO_PT_TWO_FIVE_PCT_RATE, true);
+
+        // Increase Compound v2 D3M Maximum Debt Ceiling to 20 million
+        // Set Compound v2 D3M Target Available Debt to 5 million DAI (this might already be the case)
+        DssExecLib.setIlkAutoLineParameters("DIRECT-COMPV2-DAI", 20 * MILLION, 5 * MILLION, 12 hours);
+
+        // Increase the USDP PSM tin to 0.2%
+        DssExecLib.setValue(MCD_PSM_PAX_A, "tin", 20 * WAD / 10000);   // 20 BPS
+
+
+        // MKR Transfer for CES
+        // https://vote.makerdao.com/polling/QmbNVQ1E
+
+        // CES-001 - 96.15 MKR - 0x25307aB59Cd5d8b4E2C01218262Ddf6a89Ff86da
+        GemLike(DssExecLib.mkr()).transfer(CES_WALLET, 96.15 ether); // ether as solidity alias
+
+
+        // Cage DIRECT-AAVEV2-DAI
+        // https://forum.makerdao.com/t/housekeeping-tasks-for-next-executive/19472
+        CageLike(MCD_JOIN_DIRECT_AAVEV2_DAI).cage();
+
+        // Deconstruct module for extra safety
+        DssExecLib.deauthorize(MCD_CLIP_DIRECT_AAVEV2_DAI, DOG);
+
+        // Remove module relies on end and esm so we know if our end keeper calls are out of date
+        DssExecLib.deauthorize(MCD_CLIP_DIRECT_AAVEV2_DAI, DssExecLib.end());
+        DssExecLib.deauthorize(MCD_CLIP_DIRECT_AAVEV2_DAI, DssExecLib.esm());
+        DssExecLib.deauthorize(MCD_JOIN_DIRECT_AAVEV2_DAI, DssExecLib.esm());
+
+        // Remove module from core
+        bytes32 _ilk = "DIRECT-AAVEV2-DAI";
+        DssExecLib.removeIlkFromAutoLine(_ilk);
+        (,,, uint256 _line,) = vat.ilks(_ilk);
+
+        // set core values to 0/stopped
+        DssExecLib.setValue(address(vat), _ilk, "line", 0);
+        DssExecLib.setValue(address(vat), "Line", vat.Line() - _line);
+        DssExecLib.setValue(MCD_CLIP_DIRECT_AAVEV2_DAI, "stopped", 3);
```

Comment:
So, right around here I think we may also want to remove the clipper as a reader on the oracle.  I don't know though, this might be getting a bit intractable.  

Bonus: the `clip` didn't have the `CLIPPER_MOM` filed as a ward because liquidations were disabled (that's why `stopped` is already `3` in the clip).  Othewise we may have wanted to clean that up to.

 ```suggestion
        DssExecLib.setValue(MCD_CLIP_DIRECT_AAVEV2_DAI, "stopped", 3);
        DssExecLib.removeReaderFromWhitelist(PIP_ADAI, MCD_CLIP_DIRECT_AAVEV2_DAI);
```

Reply:
So, we may not need to do this, as the `PIP_ADAI` is going to be abandoned when we redeploy.  No need to clean up the reader list.